BadHabbit/BadHabbit/Kit/NetworkKit/ProtocolType.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  ProtocolType.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 28/05/2023.
+//
+
+import Foundation
+
+enum ProtocolType: String {
```

Comment:
Confusing name. Protocol we also use to declare interfaces. Try to come out with more suitable name

Reply:
Resolved